MCprep_addon/spawner/mcmodel.py

```diff
@@ -238,13 +258,39 @@ def read_model(context: Context, model_filepath: Path) -> Tuple[Element, Texture
 	return elements, textures
 
 
-def add_model(model_filepath: Path, obj_name: str="MinecraftModel") -> bpy.types.Object:
+def add_model(model_filepath: Path, obj_name: str="MinecraftModel") -> Tuple[int, bpy.types.Object]:
 	"""Primary function for generating a model from json file."""
-	mesh = bpy.data.meshes.new(obj_name)  # add a new mesh
-	obj = bpy.data.objects.new(obj_name, mesh)  # add a new object using the mesh
-
 	collection = bpy.context.collection
 	view_layer = bpy.context.view_layer
+	
+	# Forcing usage of "fixed" block to meshswap spawner instead
+	# Theses blocks certainly are already in meshswap
+	blocks = ["chest", "banner", "lantern", "campfire"]
```

Comment:
If the user imports a JSON model of one of these blocks over using meshswap, then we **should not** decide to use the meshswap model over the JSON model. I shouldn't have to say this, but here we are.